apis.go

```diff
@@ -65,12 +71,20 @@ type OVNDBApi interface {
 	LRPAdd(lr string, lrp string, mac string, network []string, peer string, external_ids map[string]string) (*OvnCommand, error)
 	// Delete LRP with given name on given lr
 	LRPDel(lr string, lrp string) (*OvnCommand, error)
+	// Add LB to LR
+	LRLBAdd(lr string, lb string) (*OvnCommand, error)
+	// Delete LB from LR
+	LRLBDel(lr string, lb string) (*OvnCommand, error)
+	// List Load balancers for a LR
+	LRLBList(lr string) ([]*LoadBalancer, error)
 	// Add LB
 	LBAdd(name string, vipPort string, protocol string, addrs []string) (*OvnCommand, error)
 	// Delete LB with given name
 	LBDel(name string) (*OvnCommand, error)
 	// Update existing LB
 	LBUpdate(name string, vipPort string, protocol string, addrs []string) (*OvnCommand, error)
+	// List existing LBs
+	LBList() (*OvnCommand, error)
```

Comment:
why this return *OvnCommand? as i see before my prs list returns cached data and not create request to ovsdb.

Reply:
Yeah sorry typo. Forgot to add UT for that. Should have captured.

Reply:
Got rid of it!